packages/collector/test/apps/agentStubControls.js

```diff
@@ -43,6 +43,9 @@ class AgentStubControls {
         env.KAFKA_TRACE_CORRELATION = opts.kafkaConfig.traceCorrelation.toString();
       }
     }
+    if (opts.ignoreEndpoints) {
+      env.IGNORE_ENDPOINTS = JSON.stringify(opts.ignoreEndpoints);
```

Comment:
Ah yeah sorry, I see now why we need this. We need to stringify "{ redis: 'set|get' }".